src/mkl_pardiso.jl

```diff
@@ -13,7 +13,7 @@ mutable struct MKLPardisoSolver <: AbstractPardisoSolver
 end
 
 function MKLPardisoSolver()
-    if !MKL_jll.is_available()
+    if !( LOCAL_MKL_FOUND || MKL_jll.is_available())
```

Comment:
use `mkl_is_available()` here?

Reply:
Sure, sorry did not look for all occurences.